build.proj

```diff
@@ -77,4 +85,17 @@
     <RemoveDir Directories="$(BinDir)" />
   </Target>
 
+  <PropertyGroup>
+    <UserLocalFolder Condition="'$(OsEnvironment)'!='Unix'">$(LocalAppData)/</UserLocalFolder>
+    <UserLocalFolder Condition="'$(OsEnvironment)'=='Unix'">$(HOME)/.local/share/</UserLocalFolder>
+  </PropertyGroup>
+
+  <Target Name="CleanPackages">
+    <RemoveDir Directories="$(PackagesDir)" />
+  </Target>
+
+  <Target Name="CleanPackagesCache">
+    <RemoveDir Directories="$(UserLocalFolder)NuGet/Cache/;$(UserLocalFolder)NuGet/v3-cache/;$(UserLocalFolder)dnu/cache/" />
+  </Target>
```

Comment:
https://github.com/dotnet/buildtools/pull/394 puts these clean targets in BuildTools.  Is putting them here in CoreFX intended as a temporary solution until that PR is in?  I'm ready to merge that PR as soon as people sign off, so I'm not sure I see the value in that.